README.md

```diff
@@ -83,24 +86,53 @@ Online demo: https://dowjones.github.io/react-dropdown-tree-select/#/story/simpl
 
 ## Install
 
-```
-> npm i react-dropdown-tree-select
+### As NPM package
+
+```js
+npm i react-dropdown-tree-select
 
 // or if using yarn
+yarn add react-dropdown-tree-select
+```
+
+To start using it, import `react-dropdown-tree-select` and its styles in your application:
```

Comment:
You already have a "Usage" part with this code

Reply:
Good catch. This bit does feel redundant.